src/main/java/org/graylog2/log4j2/GelfAppender.java

```diff
@@ -41,7 +41,7 @@
 @Plugin(name = "GELF", category = "Core", elementType = "appender", printObject = true)
 public class GelfAppender extends AbstractAppender {
     private static final long serialVersionUID = 4796033328540158817L;
-
+    private static final String REGEX_IP_ADDRESS = "\\d+(\\.\\d+){3}";
```

Comment:
Checking if [`InetAddress#getByName(java.lang.String)`](https://docs.oracle.com/javase/8/docs/api/java/net/InetAddress.html#getByName-java.lang.String-) succeeds might be sufficient.